File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/plus_api_result_list_currency_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PlusApiResultListCurrencyVO:
    """API调用结果"""
    data: List[CurrencyVO]
    code: str
    msg: str
    request_id: str
    ip: str = None
    hostname: str = None
    error_name: str

File: sdkwork-app-sdk-kotlin/src/main/kotlin/com/sdkwork/app/PlusApiResultListCurrencyVO.kt
package com.sdkwork.app

data class PlusApiResultListCurrencyVO(
    val data: List<CurrencyVO>? = null,
    val code: String? = null,
    val msg: String? = null,
    val requestId: String? = null,
    val ip: String? = null,
    val hostname: String? = null,
    val errorName: String? = null
)
